articles/introduction-to-neural-networks/index.md

```diff
@@ -0,0 +1,208 @@
+
+Weight is a value in a neural network that converts input data within the network's hidden layers. Weights work by input layer, taking input data, and multiplying it by the weight value. It then initiates a value for the first hidden layer. The hidden layers transform the input data and pass it to the next layer. The output layer produces the desired output.
+
+The inputs and the weights are multiplied, and their sum is sent to neurons in the hidden layer. Each neuron adds the inputs it receives to get the sum. This value then transits through the activation function.
+
+The activation function outcome decides if a neuron is activated or not. An activated neuron transfer information in the other layers through the channels. In this approach, the data gets generated on the network. In other words, this is [forward propagation](https://towardsdatascience.com/forward-propagation-in-neural-networks-simplified-math-and-code-version-bbcfef6f9250). Feed-forward propagation is the process of inputting data in an input node and getting a predicted output through the output node.
+
+Feed-forward propagation takes place when the hidden layer accepts the input data. Process it as per the activation function and pass it to the output. The neuron in the output layer with a higher probability projects the result.
+
+### Main Types of Neural Networks
+
+Neural networks are identified based on mathematical performance and principles to determine the output.
+
+For more types of neural networks, [check this article out.](https://medium.com/towards-artificial-intelligence/main-types-of-neural-networks-and-its-applications-tutorial-734480d7ec8e)
+
+#### Perceptron
+
+[Minsky and Papert](https://www.researchgate.net/publication/3081582_Review_of_&#39;Perceptrons_An_Introduction_to_Computational_Geometry&#39;_Minsky_M_and_Papert_S_1969) proposed the Perceptron model (Single-layer neural network). They said it is modeled after the functioning of the human brain. It is one of the simplest models that can learn and solve complex data problems in [neural networks](https://www.mygreatlearning.com/blog/types-of-neural-networks/).
+
+Perceptron network comprises of two layers:
+
+- [Input Layer](https://www.techopedia.com/definition/33262/input-layer-neural-networks)
+- [Output Layer](https://www.techopedia.com/definition/33263/output-layer-neural-networks)
+
+The input layer computes the weighted input for every node. The input layer computes the weighted input for every node. The activation function is pertained to get the result as output.
```

Comment:
2nd sentence: "The activation function is pertained to get " 

I think this was meant to say pretrained? Not sure

Reply:
This wasn't addressed either @lalith1403 @JudyNduati FYI